velox/exec/RowContainer.h

```diff
@@ -1180,6 +1188,19 @@ class RowContainer {
   // Free any aggregates associated with the 'rows'.
   void freeAggregates(folly::Range<char**> rows);
 
+  void freeNextRowVectors(folly::Range<char**> rows) {
+    if (!nextOffset_) {
+      return;
+    }
+    for (auto row : rows) {
+      auto& vector = getNextRowVector(row);
```

Comment:
What if we only want to delete the second row in the same key but not the first?  If that is not supported, we should have some check that when we erase rows, we receive all the rows in one slot and no missing.

Reply:
Thanks, I added the check in this fucntion to make sure all rows with same key are erased in single call.